### PR TITLE
Add primary and title option to Actions

### LINKED ISSUE #1055: Ability to add text next to icon in action button component
For the Forms app, we need to supply text next to an action button, namely "Add question", as otherwise it’s not really obvious what it does:
form add question button

We are using the Actions component here, but there seems to be no way to do that. https://nextcloud-vue-components.netlify.app/#/Components/Actions

### PR DESCRIPTION
**Pull request by skjnldsv**
fix #1055

Capture d’écran_2020-04-29_14-20-14

### CODE REVIEW

**Comment by raimund-schluessler** [comment]
Nice 👍 

@georgehrke I guess this could also be used in various places of the Calendar app e.g. for the `New event` and `Today` buttons:
Screenshot_2020-04-29 Woche 24 aus 2020 - Kalender - Nextcloud

File: src/components/Actions/Actions.vue [hunk 1]
```diff
@@ -83,15 +111,21 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
...
 		<!-- If more than one action, create a popovermenu -->
-		<a class="icon action-item__menutoggle"
-			:class="defaultIcon"
+		<button class="icon action-item__menutoggle"
+			:class="{
+				[defaultIcon]: true,
+				'action-item__menutoggle--with-title': menuTitle,
+				'action-item__menutoggle--primary': primary
+			}"
 			href="#"
```
[the remark this answers is not in the document]
Comment (skjnldsv) [reply src/components/Actions/Actions.vue:120]: Oups! :see_no_evil:

[the remark this answers is not in the document]
Comment (skjnldsv) [reply src/components/Actions/Actions.vue]: Fixed!

**Comment by skjnldsv** [comment]
- [ ]  Text in our buttons is always bold, especially in the primary actions

By @jancborchardt

**Pull request merged**